app/mailers/favorite_mailer.rb

```diff
@@ -12,4 +12,14 @@ def new_comment(user, post, comment)
 
     mail(to: user.email, subject: "New comment on #{post.title}")
   end
+
+  def new_post(post)
+    headers["Message-ID"] = "<posts/#{post.id}@http://localhost:3000/>"
+    headers["In-Reply-To"] = "<post/#{post.id}@http://localhost:3000/>"
+    headers["References"] = "<post/#{post.id}@http://localhost:3000/>"
```

Comment:
This is weird, do they explain in the video why they are changing the request object in the controller. I could understand if they were changing the [response object](http://guides.rubyonrails.org/action_controller_overview.html#the-response-object).